Apps/Admin/Server/Controllers/AgentAccessController.cs

```diff
@@ -73,20 +74,21 @@ public async Task<AdminAgent> ProvisionAgentAccess(AdminAgent agent)
         /// <param name="query">The query string to match agents against.</param>
         /// <returns>The collection of matching agents.</returns>
         /// <response code="200">Returns the collection of matching agents.</response>
-        /// <response code="401">The client must authenticate itself to get the requested response.</response>
+        /// <response code="401">The client must authenticate itself to get the requested resource.</response>
         /// <response code="403">
         /// The client does not have access rights to the content; that is, it is unauthorized, so the server
         /// is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.
         /// </response>
         /// <response code="502">Unable to get response from Keycloak.</response>
         [HttpGet]
-        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<AdminAgent>))]
```

Comment:
This can be made explicit by changing the return type to `Task<ActionResult<AdminAgent>>`